.github/workflows/ci-conda.yml

```diff
@@ -105,14 +107,15 @@ jobs:
         run: pip check
 
       - name: Test
+        if: always()
```

Comment:
I think this should be `success() or failure()`

Reply:
What's the difference?

Reply:
so that in a canceled run, no new test is started.

Reply:
Okay, done.